src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeInstruct.java

```diff
@@ -3,31 +3,27 @@
 public class HomeOfficeInstruct {
 
     private ConsumerReference consumerReference;
-    private CourtOutcome courtOutcome;
```

Comment:
CourtOutcome is required for other notification. Why remove it if it was already existing?

Reply:
It's been restructured for re-usability. This is only for common attributes and specific fields will be used on the required events and message types

Reply:
ok